lua/gitlab/actions/discussions/winbar.lua

```diff
@@ -49,9 +49,17 @@ local function content()
   local resolvable_notes, resolved_notes = get_data(state.DISCUSSION_DATA.unlinked_discussions)
 
   local draft_notes = require("gitlab.actions.draft_notes")
-  local inline_draft_notes = List.new(state.DRAFT_NOTES):filter(draft_notes.has_position)
-  local unlinked_draft_notes = List.new(state.DRAFT_NOTES):filter(function(note)
-    return not draft_notes.has_position(note)
+  local inline_draft_notes, unlinked_draft_notes = List.new(state.DRAFT_NOTES):partition(function(note)
+    if note.discussion_id == "" then
+      return draft_notes.has_position(note)
+    else
+      for _, discussion in ipairs(state.DISCUSSION_DATA.unlinked_discussions) do
```

Comment:
I iterate over `unlinked_discussions` here because, in my experience, there are fewer of them in a typical MR.